src/firefly/js/core/JsonUtils.js

```diff
@@ -83,7 +83,7 @@ export const jsonRequest= function(baseUrl, cmd, paramList, doPost) {
                 return;
             }
             response.json().then( (result) => {
-                if (has(result,'0')) {
+                if (has(result,'0.success')) {
```

Comment:
I think I just changed it back to how it used to be before 10/21.

As we talked today, this code is handling JSON wrapper of possibly non-JSON result. The wrapping is done in ServCommand processRequest:33 and 53 and the wrapper always has 'success' field set either to true or false. Hence, checking has(result,'0.success') is a way to tell if JSON result is wrapped.

The advantage of checking for 'success' field is that now we can pass arrays as JSON result.

Just a note: when we change how we wrap JSON result in ServCommand.java, we need to change jsonRequest in JsonUtil.js, namely this precise code.